modules/frontend/frontend-editors-web/src/META-INF/resources/html/editors/ckeditor_diffs/extension/creole_dialog_definition.js

```diff
@@ -0,0 +1,55 @@
+CKEDITOR.on(
+	'dialogDefinition',
+	function(event) {
+		if (event.editor === ckEditor) {
+			var dialogName = event.data.name;
+
+			var dialogDefinition = event.data.definition;
+
+			var infoTab;
+
+			if (dialogName === 'cellProperties') {
+				infoTab = dialogDefinition.getContents('info');
+
+				infoTab.remove('bgColor');
+				infoTab.remove('bgColorChoose');
+				infoTab.remove('borderColor');
+				infoTab.remove('borderColorChoose');
+				infoTab.remove('colSpan');
+				infoTab.remove('hAlign');
+				infoTab.remove('height');
+				infoTab.remove('htmlHeightType');
+				infoTab.remove('rowSpan');
+				infoTab.remove('vAlign');
+				infoTab.remove('width');
+				infoTab.remove('widthType');
+				infoTab.remove('wordWrap');
+
+				dialogDefinition.minHeight = 40;
+				dialogDefinition.minWidth = 210;
+			}
+			else if (dialogName === 'table' || dialogName === 'tableProperties') {
+				debugger;
```

Comment:
So sorry, Nate, I missed it :disappointed: 

See https://github.com/brianchandotcom/liferay-portal/pull/27501